Ricardo andre pool cen

Perla233 opened the pull request:
Aqui esta mi investigacion

ricardo-andre-pool-cen committed:
Merge pull request #48 from ricardo-andre-pool-cen/Primera_Entrega

Actualizacion de contenidos para katherine

ricardo-andre-pool-cen committed:
Create Entrega de descripcion del proyecto, propuesta de valor y algunos requisitos

documento para la entrega de contribucion

KatherineOr18 committed:
Create Requisitos funcionales 

12 requisitos funcionales

KatherineOr18 committed:
Requisitos Funcionales generalizado

Los nuevos requisitos

KatherineOr18 committed:
Requisitos Funcionales generalizado

Cambio de las funciones

KatherineOr18 committed:
Delete Requisitos funcionales

KatherineOr18 committed:
Delete Comparación entre licenciaturas, UADY y TECNM

KatherineOr18 committed:
Delete Entrega de descripcion del proyecto, propuesta de valor y algunos requisitos

KatherineOr18 committed:
Update Act#3 Requisitos Funcionales

LuisGilDzib committed:
Create Guión para la presentación del proyecto.

guión de prueba

LuisGilDzib committed:
Update Guión para la presentación del proyecto.

he actualizado y aplicado los cambios solicitados, es espera de una nueva revisión.

ricardo-andre-pool-cen committed:
Merge pull request #78 from ricardo-andre-pool-cen/Ricardo-Andre-Pool-Cen

Actualización de rama -> Aprobado, revisa las asignaciones y empieza con el trabajo.

LuisGilDzib committed:
Delete ESPAÑOL/Periodo_1/Act#1_Descripción del proyecto.md

LuisGilDzib committed:
Delete ESPAÑOL/Periodo_1/Dia 14.md

elimino este documento,  ya que no laboré el 14 XD

LuisGilDzib committed:
Bitagora del Dia 16.md

Bitagora de mi trabajo diario - Luis Dzib

LuisGilDzib committed:
Bitácora del Dia 16.md

Bitácora de mi contribución del día al proyecto.

LuisGilDzib committed:
Bitácora del Dia 16.md

Refinamiento del diseño de la descripción de mi contribución diaria.

LuisGilDzib committed:
Delete ESPAÑOL/Pre-periodo directory

no labore el 9 de octubre

LuisGilDzib committed:
Delete ESPAÑOL/Act#1 Descripción del proyecto.md

documento de la rama principal

LuisGilDzib committed:
Delete ESPAÑOL/Act#2 Administración.md

LuisGilDzib committed:
Delete ESPAÑOL/Act#3 Requisitos Funcionales.md

LuisGilDzib committed:
Delete ESPAÑOL/Act#4 Requisitos No funcionales.md

LuisGilDzib committed:
Delete ESPAÑOL/Act#6 Presentación.md

LuisGilDzib committed:
Delete ESPAÑOL/Act#5 Diseño.md

LuisGilDzib committed:
Delete ESPAÑOL/Act#7 Competencias.md

LuisGilDzib committed:
Delete Materials-Materiales directory

carpeta de la rama principal, no es esencial en mi rama

LuisGilDzib committed:
Delete README.md

LuisGilDzib committed:
Update README.md

ricardo-andre-pool-cen committed:
Merge pull request #93 from ricardo-andre-pool-cen/Ricardo-Andre-Pool-Cen

Importe de los contenidos nuevos de mi rama hacia el cuerpo del proyecto.

ricardo-andre-pool-cen committed:
Eliminado el contenido antiguo.

Causa conflictos con la solicitud de actualizacion, se eliminó para solucionar el problema.

ricardo-andre-pool-cen committed:
Merge pull request #95 from ricardo-andre-pool-cen/Ricardo-Andre-Pool-Cen

Actualización de rama - aprobado

ricardo-andre-pool-cen committed:
Removidos contenidos que causan contenido

En un momento se importaran los nuevos contenidos pero estos documentos causan conflicto, he salvado el daily del Día 16 y lo volveré a colocar manualmente

ricardo-andre-pool-cen committed:
Re-añadido este documento para solucionar problemas de importación

ricardo-andre-pool-cen committed:
Merge branch 'Luis-Gilberto-Dzib-Pech' into Cuerpo_del_proyecto

ricardo-andre-pool-cen committed:
Merge pull request #94 from ricardo-andre-pool-cen/Cuerpo_del_proyecto

reparacion de daños - aprobado

ricardo-andre-pool-cen committed:
Salvación del diario de Luis gilberto.

LuisGilDzib committed:
Create Dia 18

LuisGilDzib committed:
Update Dia 18

bitácora del día 18

LuisGilDzib committed:
Dia 18.md

Bitácora del día 18

LuisGilDzib committed:
Update Act#3 Requisitos Funcionales.md

Historias de usuario

ElPiniero committed:
Redacción del nuevo proceso con los artefactos de scrum.

Este es el avance realizado para esta actividad, en un rato se añaden mas contenidos, falta agregar las historias de usuario y la documentación especifica de los artefactos generados en el Periodo_1.

ElPiniero committed:
Merge remote-tracking branch 'origin/Ricardo-Andre-Pool-Cen' into Ricardo-Andre-Pool-Cen

KatherineOr18 committed:
Actualización Historias de usuario Informe de investigación - El nuevo proceso.md

Agrego mi investigación de los puntos relevantes sobre "Historias de usuario"

KatherineOr18 committed:
Historias de usuario, nuevo - El nuevo proceso.md

Le agregué unas referencias

KatherineOr18 committed:
Actualización Informe de investigación - El nuevo proceso.md

Investigación respecto a las historias de usuario

LuisGilDzib committed:
Update Dia 18.md

LuisGilDzib committed:
Update and rename Dia 18.md to Día 18.md

LuisGilDzib committed:
Día 18.md

LuisGilDzib committed:
Delete Día 18.md

LuisGilDzib committed:
Create Día 18.md

LuisGilDzib committed:
Día 18.md

LuisGilDzib committed:
Día 18.md

LuisGilDzib committed:
Día 18.md

Optimización de la visualización de mi bitácora.

ricardo-andre-pool-cen committed:
Merge branch 'Katherine-Aolani-Ordoñez-Pacheco' into Cuerpo_del_proyecto

LuisGilDzib committed:
Día 18.md

Bitácora del día 18.

LuisGilDzib committed:
Update Act#3 Functional requirements.md

Traducción de los requisitos al idioma inglés

LuisGilDzib committed:
Update Act#3 Functional requirements.md

Traducción de los requisitos funcionales a el idioma inglés.

LuisGilDzib committed:
Update Act#3 Functional requirements.md

Traducción de los requisitos he historias de usuarios en el idioma inglés.

LuisGilDzib committed:
Create Day 18.md

Traducción de mi bitácora al idioma inglés.

LuisGilDzib committed:
Create Day 16.md

Traducción al idioma inglés de mi bitácora día 16

LuisGilDzib committed:
Update Act#3 Requisitos Funcionales.md

Implementación de mejoras ortográficas y de visualización.

LuisGilDzib committed:
Update Día 18.md

Implementación de mejoras técnicas a mi bitácora.

LuisGilDzib committed:
Update Day 18.md

Implementación de mejoras técnicas y traducción de mi bitácora.

ricardo-andre-pool-cen committed:
Merge branch 'perla-canche' into Ricardo-Andre-Pool-Cen

ricardo-andre-pool-cen reviewed (changes requested):
## Definición de Terminado:
- [x] El artefacto es documentado en el Periodo_1, en el documento "Informe de investigación - El nuevo proceso"
- [x] Se adjunta el informe de investigación de los nuevos artefactos de **SCRUM** a utilizar mediante paráfrasis.
- [ ] Enlista las fuentes de información en formato APA 7th.
- [x] La redacción es cohesiva y correcta, con buena ortografía.

El contenido es correcto, sin embargo, aplicas mal la solicitud de importación, debes efectuarla desde tu rama hacia el cuerpo del proyecto, es decir:
base: cuerpo del proyecto
comparación: tu rama

ricardo-andre-pool-cen committed:
Daily del día 18

Se hace el informe de las actividades hechas en este día, con esto se finaliza el Periodo_1.

ricardo-andre-pool-cen committed:
Creación de la Declaración de problemas del Periodo_1.md

Preparado el documento para posterior redacción.

ricardo-andre-pool-cen committed:
Merge branch 'Ricardo-Andre-Pool-Cen' into Cuerpo_del_proyecto

ricardo-andre-pool-cen committed:
Eliminacion del documento antiguo

Voy a meter uno nuevo pero manualmente, se esta descontrolando la documentación.

ricardo-andre-pool-cen committed:
Carga del nuevo documento.

Este archivo tiene las investigaciones de todos los contribuyentes hasta el momento, y esta ordenado.

ricardo-andre-pool-cen committed:
Documento que no debía estar aquí

ricardo-andre-pool-cen committed:
Merge branch 'Cuerpo_del_proyecto' into Luis-Gilberto-Dzib-Pech

ricardo-andre-pool-cen committed:
Merge pull request #97 from ricardo-andre-pool-cen/Luis-Gilberto-Dzib-Pech

Historias de Usuario.

ricardo-andre-pool-cen committed:
Resultados del Periodo_1

Adjunto evidencia de como acabó el Periodo_1

ElPiniero committed:
Avances del dia 21 de Octubre

Repare las historias de usuario, se terminó la documentación de la investigacion.

ricardo-andre-pool-cen committed:
Agregado el enlace al video

ricardo-andre-pool-cen committed:
Pruebas de tabla con casillas de verificacion

ricardo-andre-pool-cen committed:
Pruebas de tabla con casillas de verificacion 2

ElPiniero committed:
Avances del día 23 de octubre

Se publica la primer captura de la contribución individual, en excel, aun veré como desplegarla desde Github. se arregla la documentación diaria y se incluyen los "dailys" correspondiente.

ElPiniero committed:
Merge remote-tracking branch 'origin/Ricardo-Andre-Pool-Cen' into Ricardo-Andre-Pool-Cen

LuisGilDzib committed:
Update Act#6 Presentación.md

LuisGilDzib committed:
Pre-guion Act#6 Presentación.md

Implementación del pre-guion, le falta refinamiento, se espera el refinamiento para cuando se terminen los artefactos restantes para llenar la fase 2 y fase 3 de esta propuesta de guion.

LuisGilDzib committed:
Pre-guion Act#6 Presentación.md

Mi sugerencia para el guion de la presentación, se espera un refinamiento de este guion cuando se termine el resto de los artefactos.

ricardo-andre-pool-cen committed:
Merge branch 'Cuerpo_del_proyecto' into Ricardo-Andre-Pool-Cen

ricardo-andre-pool-cen committed:
Documento antiguo

ricardo-andre-pool-cen committed:
Documento antiguo

LuisGilDzib committed:
Update Act#6 Presentación.md

Pre-guion que incluye una generalización de las ideas en como pueden ir las escenas, la información, estructura del guion para el video y sugerencias de ritmo.

LuisGilDzib committed:
Update Act#6 Presentation.md

English version of the script for the presentation

LuisGilDzib committed:
Update Act#3 Requisitos Funcionales.md

Refinamiento de las historias de usuario

LuisGilDzib committed:
Update Act#3 Functional requirements.md

refinement of user stories.

LuisGilDzib committed:
Update Act#3 Functional requirements.md

refinement of user stories.

LuisGilDzib committed:
Update Act#3 Functional requirements.md

refinement of user stories.

LuisGilDzib committed:
Bitácora del Día 24.md

Bitácora del día 24

LuisGilDzib committed:
Update Día 24.md

links correctos a los documentos mencionados.

LuisGilDzib committed:
Binnacle Day 24.md

Log of the 24th.

ElPiniero committed:
Arreglos del día 24 de Octubre.

Ocurrió un fallo con el movimiento de archivos de los "daily", mi rama en la nube se vio afectada, vuelvo a actualizar el contenido.

ElPiniero committed:
Merge remote-tracking branch 'origin/Ricardo-Andre-Pool-Cen' into Ricardo-Andre-Pool-Cen

ricardo-andre-pool-cen committed:
Merge pull request #104 from ricardo-andre-pool-cen/Luis-Gilberto-Dzib-Pech

Guión y actualización de Historias de Usuario.

ricardo-andre-pool-cen commented:
Solicitud antigua
